### DISCUSSION TITLE
Themes for Quasar

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9550] [General - Components / Directives / etc] [answered]
Hey community!

Although the default theme by Quasar is great, I would like to ask whether there are additional themes for apps? Is it possible to customize Quasar in a way that the app would look different from the "default" theme?

**Comment by hawkeye64** [comment]
@i-mage Any theming also takes a lot of CSS know-how. Here is my attempt to do something completely different. The attempt here, was to make it look like an admin panel, and not material design: https://codepen.io/Hawkeye64/pen/WNNdKJV

**Reply by i-mage** [reply]
That looks good! Thanks!

**Comment by ontwikkelfabriek** [comment, marked as answer]
@i-mage 
>I would like to ask whether there are additional themes for apps

No, there used to be an IOS theme but that's long gone unfortunately. 

>  Is it possible to customize Quasar in a way that the app would look different from the "default" theme?

Yes it's possible but you'll need some serious css skills and time.

@hawkeye64 

Ah I just wanted to post your codepen to show off what you can do with styling. Btw are there any (long time) plans for theming in Quasar? Or support/css-compatibility for Tailwind 2 components?

**Reply by hawkeye64** [reply]
@ontwikkelfabriek 

Yes, we have talked about "naked" components (property `naked`, or `plain`, etc) where Users will be 100% in charge of UI styling. Likely to be in Quasar v3.

Also, we have talked about themes. Free and chargeable ones (similar to what other UI Libraries do). Again, likely Quasar v3.

As an aside, these may-or-may-not happen. I think we need to do some POC first.

**Reply by i-mage** [reply]
Great to hear that. I really like Quasar so far, an excellent contribution to the Vue ecosystem. Thank you.
